plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -83,6 +94,17 @@ del_ovs_flows() {
     ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_src=${ipaddr}"
     ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_dst=${ipaddr}"
     ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_src=${ipaddr}"
+
+    qos=$(ovs-vsctl list port ${veth_host} >&/dev/null | grep qos | cut -d ":" -f 2)
```

Comment:
You probably copied that from somewhere else but there's a cleaner way to do it:

    ovs-vsctl get Port ${veth_host} qos